<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myDemo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/images/cookie.bmp</normaloff>:/icons/images/cookie.bmp</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/ice.ico</normaloff>:/icons/images/ice.ico</iconset>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="plainText">
    <string>This is a plain text edit widget!!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>226</width>
     <height>64</height>
    </rect>
   </property>
   <property name="title">
    <string>Font</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_Underline">
      <property name="text">
       <string>Underline</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Italic">
      <property name="text">
       <string>Italic</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Bold">
      <property name="text">
       <string>Bold</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>231</width>
     <height>64</height>
    </rect>
   </property>
   <property name="title">
    <string>Color</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_Red">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_Green">
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_Blue">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Reset">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/cola.png</normaloff>:/icons/images/cola.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Submit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/turkey.png</normaloff>:/icons/images/turkey.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Reset</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
